messages/diagnostics.json

```diff
@@ -0,0 +1,7 @@
+{
+  "apiVersionMismatch": "The sourceApiVersion in sfdx-project.json does not match the apiVersion. This means deploying and retrieving source will use sourceApiVersion. This may or may not be desired.",
+  "apiVersionUnset": "Both sourceApiVersion and apiVersion are not defined. This means deploying and retrieving source will use the max apiVersion of the target org. This may or may not be desired.",
+  "maxApiVersionMismatch": "The max apiVersion for the default DevHub org does not match the max apiVersion for the default target org. This means the default target orgs are running different versions and without explicitly setting an apiVersion there may be deploy/retrieve issues.",
```

Comment:
```suggestion
  "maxApiVersionMismatch": "The max apiVersion of the default DevHub org doesn't match the max apiVersion of the default target org. This mismatch means that the default target orgs are running different API versions. Be sure you explicitly set the apiVersion when you deploy or retrieve source, or you will likely run into problems.",
```

Reply:
I changed the suggestion into a positive (i.e. "set the apiVersion") rather than a negative.  Okay?